src/components/lesson/data.ts

```diff
@@ -1,17 +1,50 @@
+
+export type FormItemData = {
+  itemId?: string;
+  displayName?: string | React.ReactNode;
```

Comment:
We should only use serializable types in our data models (if you need React.ReactNode you should define it in the component properties). Consider data as something (that can be persisted and retrieved). For example,

`
type ComponentProps = ComponentData & {
   children?: React.ReactNode
   myCallback?: () => void;
}
`